types/truffle-hdwallet-provider/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for truffle-hdwallet-provider 0.0
+// Project: https://github.com/trufflesuite/truffle-hdwallet-provider#readme
+// Definitions by: Leonid Logvinov <https://github.com/LogvinovLeon>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+import {
+    JSONRPCRequestPayload,
+    JSONRPCResponsePayload,
+    Provider
+} from "ethereum-protocol";
+declare class HDWalletProvider implements Provider {
+    constructor(mnemonic: string, rpcUrl: string);
```

Comment:
You want to have third parameter address_index?: number as well as per https://github.com/trufflesuite/truffle-hdwallet-provider#general-usage

Reply:
Added it